lisa/sut_orchestrator/azure/features.py

```diff
@@ -80,3 +87,45 @@ def _is_supported(self) -> bool:
         if not isinstance(self._node.os, supported_distro):
             return False
         return True
+
+
+class Sriov(AzureFeatureMixin, features.Sriov):
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        super()._initialize(*args, **kwargs)
+        self._initialize_information(self._node)
+
+    def _switch_sriov(self, platform: Platform, enable: bool) -> None:
+        network_client = get_network_client(platform)
+        compute_client = get_compute_client(platform)
+        node = compute_client.virtual_machines.get(
```

Comment:
Don't use "node" as name here, it's confused with "node" in LISA. Because there is the Azure platform already, it can be called as vm.